src/components/EmptyState/EmptyState.test.tsx

```diff
@@ -0,0 +1,45 @@
+import { render } from "@testing-library/react";
+import React from "react";
+
+import EmptyState from "./EmptyState";
+
+describe("EmptyState ", () => {
+  it("renders the title", () => {
+    const { container } = render(
+      <EmptyState title="Test title" image={<img alt="" src="" />} />
+    );
+    expect(container).toContainHTML("Test title");
+  });
+
+  it("renders the image", () => {
+    const { container } = render(
+      <EmptyState
+        title="Test title"
+        image={<img alt="" src="path/to/image" />}
+      />
+    );
+    expect(container).toContainHTML("path/to/image");
```

Comment:
This could be something like `expect(screen.getByRole("img")).toHaveAttribute("src", "path/to/image");`